compiler/src/dmd/backend/dtype.d

```diff
@@ -1192,7 +1192,6 @@ uint param_t_length(scope param_t* p)
  * Input:
  *      ptali   initial template-argument-list
  */
-
 @trusted
```

Comment:
did you mean to remove this?

Reply:
I removed it, found out it didn't work when compiling without dip1000, and restored it, but forgot the extra whitespace.